Makefile.am

```diff
@@ -167,6 +166,16 @@ ctags_LDADD += $(SECCOMP_LIBS)
 ctags_LDADD += $(ICONV_LIBS)
 dist_ctags_SOURCES = $(CMDLINE_HEADS) $(CMDLINE_SRCS)
 
+if HOST_MINGW
+WINDRES = windres
+RES_OBJ = win32/ctags.res.o
+ctags_LDADD += $(RES_OBJ)
+$(RES_OBJ): win32/ctags.rc win32/ctags.exe.manifest win32/resource.h
+	mkdir -p $(builddir)/win32
+	$(WINDRES) -o $@ -O coff $<
+MOSTLYCLEANFILES = $(RES_OBJ)
```

Comment:
I prefer to
```
MOSTLYCLEANFILES += $(RES_OBJ)
```

To do as I write, a line 
```
MOSTLYCLEANFILES =
```
may be needed somewhere at the head of Makefile.am.

Reply:
If so, maybe `CLEANFILES` needs to be changed in a similar way?

At the head of Makefile.am:

```
CLEANFILES =
```

and at the middle of the file:

```
CLEANFILES += $(REPOINFO_HEADS) $(PEG_SRCS) $(PEG_HEADS)
```

Reply:
I changed both `CLEANFILES` and `MOSTLYCLEANFILES`.

Reply:
Yes. You have already done. Thank you.